CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+1.0.3 (2017-02-14)
+------------------
+# Out of Beta!
+After about a year and a half of hard work, we are proud to call this a 1.0 release. There is still work to do, but Realm Xamarin is now being used by thousands of developers and has proven reliable.
+
+## Sync
+Realm Xamarin now works with the Realm Mobile Platform. This means that you can write Xamarin apps that synchronize seamlessly with a Realm Object Server, allowing you to write complex apps with Xamarin that are offline-first and automatically synchronised literally by adding a few lines of configuration code.
+You can read about this in the [documentation](https://realm.io/docs/xamarin/latest/#sync)
+
+## Windows Desktop
+Realm Xamarin is no longer iOS and Android only. You can now use it to write .NET programs for Windows Desktop. Add the NuGet package to your regular .NET project and start using Realm. Sync does not yet work for Windows, but we are working on it and you can expect support soon.
+
+### Breaking Changes
+ - `IRealmCollection<T>.ObjectSchema` is deprecated and replaced with `ISchemaSource.ObjectSchema`. (#1216) 
+
+### Bug fixes
+ - `[MapTo]` attribute is now considered in queries (#1219)
+ - Letting a Realm be finalized instead of disposing it will no longer lead to crashes (#1212)
+ - Unsubscribing from PropertyChanged in a PropertyChanged callback should no longer lead to crashes (#1207)
+ - `WriteAsync` now advances the read transaction (#1192)
```

Comment:
This sounds a bit obscure, maybe add _ensuring other threads see the effects of the Write_ ?